{{ cookiecutter.repo_name }}/docs/structure/README.md

```diff
@@ -18,29 +18,36 @@ structure is explained here:
 Each subsection here contains a brief description about the files at the top-level of
 this Git repository.
 
-### `.envrc`
+### `.env`
 
-A file containing environment variables for the Git repository that can be selectively
-loaded. [`.envrc` uses the `direnv` shell extension to load these environment
-variables][direnv].
+Environment variables go here, and can be read in by Python using the `python-dotenv` package, and `os.getenv`:
 
-This file contains a `sed` command to output a `.env` file with all the environment
-variables. This may be useful for sourcing environment variables, for example in
-conjunction with PyCharm's EnvFile plugin.
+```in script
+from dotenv import load_dotenv
```

Comment:
Is there a reason for using load_dotenv vs os?

Reply:
You use both. You use load_dotenv to import the .env file and then os.getenv to output a specific value from .env